docs/kwil-cli/installation.md

```diff
@@ -0,0 +1,20 @@
+---
+sidebar_position: 1
+sidebar_label: "Installation"
+id: kwil-cli-installation
+title: Installation
+description: Install kwil-cli
+slug: /kwil-cli/installation
+---
+
+While the `kwil-cli` is closed source, you can download the binary [here](https://github.com/kwilteam/kwil-release-repo/releases/tag/v0.6.0).
+
+There are binaries for the following platforms:
+
+* Linux amd64
+* Linux arm64
+* Darwin (MacOS) Universal (amd64+arm64)
+
+After downloading the binary for your platform, add the binary file path to the PATH environment variable.
```

Comment:
Should we add an example or link for how to export to path?

Reply:
Probably not.  This is super standard stuff